test-framework/sudo-test/src/lib.rs

```diff
@@ -478,7 +480,6 @@ pub fn TextFile(contents: impl AsRef<str>) -> TextFile {
 
 impl TextFile {
     const DEFAULT_CHMOD: &'static str = "000";
```

Comment:
Very minor point, but maybe it's good to keep the `DEFAULT_CHMOD` and `DEFAULT_CHOWN` constants here, but turn them into `String` instead; instead of inlining the `DEFAULT_CHOWN` below.

Reply:
Ok, I understand that that would require the introduction of `LazyCell` here, and that would be a bit overkill.